packages/storefront-lwc/src/modules/commerce/data/cart.js

```diff
@@ -0,0 +1,225 @@
+/*
+    Copyright (c) 2020, salesforce.com, inc.
+    All rights reserved.
+    SPDX-License-Identifier: BSD-3-Clause
+    For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+*/
+/**
+ * A cart service to add to cart, load cart and blast off events
+ */
+import { apiClient } from '../api/client';
+import gql from 'graphql-tag';
+class Cart {
+    cart = {};
+
+    listeners = [];
+
+    isCartLoaded = false;
+
+    /**
+     * Calling Add to the cart BFF.
+     * @param product: the product to add to Cart
+     */
+
+    // TODO : wire up the UI quantity selector to pass in quantity to add
+    addToCart(product, qty) {
```

Comment:
We should use the lwc-apollo-client instead

Reply:
Will be done in the next iteration